splitio/push/splitworker.py

```diff
@@ -75,6 +76,8 @@ def _run(self):
                         new_split = from_raw(json.loads(self._get_feature_flag_definition(event)))
                         if new_split.status == Status.ACTIVE:
                             self._feature_flag_storage.put(new_split)
+                            self._telemetry_runtime_producer.record_update_from_sse()
```

Comment:
you must to record the telemetry outside of the `if else`, because you are updating the storage in both scenarios.

Reply:
will fix